installation/macos.md

```diff
@@ -73,28 +74,27 @@ From inside openHAB's root directory (e.g. `~/openhab/), run openHAB's startup s
 
 The first run may take a while to process, wait until the "`openhab>`" prompt appears, for example:
 
-```shell_session
-user:openhab $ sudo ./start.sh
+```shell
+$ ./start.sh
 Launching the openHAB runtime...
 
-                          __  _____    ____
-  ____  ____  ___  ____  / / / /   |  / __ )
- / __ \/ __ \/ _ \/ __ \/ /_/ / /| | / __  |
-/ /_/ / /_/ /  __/ / / / __  / ___ |/ /_/ /
-\____/ .___/\___/_/ /_/_/ /_/_/  |_/_____/
-    /_/                        2.0.0 ...
-
+                           _   _     _     ____  
+   ___   ___   ___   ___  | | | |   / \   | __ ) 
+  / _ \ / _ \ / _ \ / _ \ | |_| |  / _ \  |  _ \ 
+ | (_) | (_) |  __/| | | ||  _  | / ___ \ | |_) )
+  \___/|  __/ \___/|_| |_||_| |_|/_/   \_\|____/ 
+       |_|       3.0.0 - Release Build
 
-Hit '<tab>' for a list of available commands
+Use '<tab>' for a list of available commands
 and '[cmd] --help' for help on a specific command.
-Hit '<ctrl-d>' or type 'system:shutdown' or 'logout' to shutdown openHAB.
+To exit, use '<ctrl-d>' or 'logout'.
+
 
 openhab>
 ```
 
-Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorials/beginner/1sttimesetup.html).
-
-![](images/Accueil_Openhab_2.png)
+Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorials/first_steps.html). 
```

Comment:
I changed the link to first_steps.html, because in preview I got a 404

Reply:
```suggestion
Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorial/first_steps.html). 
```

on letter too much. :)

Reply:
Ah, the folder changed too. Thanks